### PR TITLE
Kt feature loader

### PR DESCRIPTION
**Pull request by kyltec**

### CODE REVIEW

File: client/js/components/Loader/styles.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    width: "100%"
+  },
+  loadingText: {
+    color: colors.white
+  },
+  textContainer: {
+    justifyContent: "center",
+    flex: 1,
+    alignItems: "center",
+    marginTop: 300
```
Comment (siddharthparmar7) [inline client/js/components/Loader/styles.js]: you should not have this much margin. Please use flexbox properties instead.

File: client/js/screens/Rewards/RewardsContainer.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import gql from "graphql-tags";
+import { Query } from "React-apollo";
+
+import Rewards from "./Rewards";
+
+export default class RewardsContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```
Comment (siddharthparmar7) [inline client/js/screens/Rewards/RewardsContainer.js]: if you are not creating a state then remove the constructor.

File: client/js/screens/Rewards/RewardsContainer.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+  render() {
+    <Query
```
Comment (siddharthparmar7) [inline client/js/screens/Rewards/RewardsContainer.js]: write a proper <Query> tag.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by kyltec** [comment]
added asset files on this branch

File: client/js/components/Loader/Loader.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        />
+
+        <Text style={styles.loadingText}>Pouring...</Text>
+      </View>
+    );
+  }
+}
+
+Loader.propTypes = {
+  styles: PropTypes.object.isRequired
```
Comment (siddharthparmar7) [inline client/js/components/Loader/Loader.js]: styles is not a prop. It a const that you imported from your styles.js file. Please remove proptypes if you do not have props to validate.

File: client/js/screens/Card/CardContainer.js [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
+import React, { Component } from "react";
+import { View } from "react-native";
 
-import Card from './Card';
-import styles from './styles';
+import Card from "./Card";
+import styles from "./styles";
 
 class CardContainer extends Component {
   constructor(props) {
```
Comment (siddharthparmar7) [inline client/js/screens/Card/CardContainer.js]: If you do not have a state remove the constructor.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**